src/CodeReviews.coffee

```diff
@@ -137,14 +137,20 @@ class CodeReviews
   # @param String origin_room string of origin room
   # @param String channel_to_notify string of the user/room to notify
   notify_channel: (cr, origin_room, channel_to_notify) ->
-    attachments = []
-    attachments.push
-      fallback: "#{cr.url} could use your :eyes: Remember to claim it in ##{origin_room}"
-      text: "*<#{cr.url}|#{cr.slug}>* could use your :eyes: Remember to claim it" +
-      " in <https://slack.com/app_redirect?channel=#{origin_room}|##{origin_room}>"
-      mrkdwn_in: ["text"]
-      color: "#575757"
-    sendFancyMessage @robot, channel_to_notify, attachments
+    slackTeamId(@robot).then (teamId) ->
+      if teamId?
+        channel_uri = "slack://app?team=#{teamId}&id=#{channel_to_notify}"
```

Comment:
The generated link looks like [this](https://alleyinteractive.slack.com/archives/C23NDCLGG/p1664299343835419)

![Screen Shot 2022-09-27 at 11 24 13 AM](https://user-images.githubusercontent.com/7146063/192594548-6f718b51-c479-42e4-8f22-1e1bd28e9cf6.png)

instead of linking to the origination channel, it is linking to the destination channel

Reply:
fixing that now